crates/noirc_frontend/src/graph/mod.rs

```diff
@@ -50,7 +49,11 @@ pub struct CrateGraph {
 
 impl CrateGraph {
     pub fn is_last_crate(&self, crate_id: CrateId) -> bool {
-        (self.arena.len() - 1) == crate_id.0
+        match crate_id {
+            CrateId::Crate(crate_id) | CrateId::Stdlib(crate_id) => {
```

Comment:
```suggestion
        let crate_id = crate_id.id();
```

Add a method for this? Although we need some clarification on what this inner id means.

Reply:
With this change, the "crate id" is the entirety of the type and usize id and they don't make sense separated. This match is fine